share/zm-backend.sh

```diff
@@ -37,7 +37,7 @@ start() {
 stop() {
     if [ -f $PIDDIR/zonemaster_backend_testagent.pid ]
     then
-        $BASEDIR/bin/zonemaster_backend_testagent --user=$USER --group=$GROUP --pidfile=$PIDDIR/zonemaster_backend_testagent.pid stop
+        $BASEDIR/bin/zonemaster_backend_testagent --logfile=$LOGDIR/zonemaster_backend_testagent.log --user=$USER --group=$GROUP --pidfile=$PIDDIR/zonemaster_backend_testagent.pid stop
```

Comment:
Is this really correct? I would expect the log file to be stated at start, not at stop.

Reply:
It looks a bit weird but it is in fact correct. From what I can tell we're not making use of it at the moment, but Daemon::Control (which we're using) provides hooks for the stop command, and if add logging in such a hook, we want the log lines to end up in the correct log file.
However, of course the logfile argument should be included in the start command as well. So I added that.